cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -88,7 +96,7 @@ function configure_proxy_flags() {
     elif ! [[ "${metastore_instance}" =~ .+:.+:.+ ]]; then
       err 'hive-metastore-instance must be of form project:region:instance'
     elif ! [[ "${metastore_instance}" =~ =tcp:[0-9]+$ ]]; then
-      metastore_instance+="=tcp:${metastore_proxy_port}"
+      metastore_instance+="=tcp:0.0.0.0:${metastore_proxy_port}"
```

Comment:
Why is this necessary?

Reply:
it was needed because otherwise it'll just listen on localhost, and the other machines won't be able to connect to it. But it's no longer needed now.